tests/phpunit/integration/Core/Util/Build_ModeTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Build_ModeTest
+ *
+ * @package   Google\Site_Kit\Tests\Core\Util
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Util;
+
+use Google\Site_Kit\Core\Util\Build_Mode;
+use Google\Site_Kit\Tests\TestCase;
+use ReflectionMethod;
+
+class Build_ModeTest extends TestCase {
+
+	public function test_get_mode() {
+		$method = new ReflectionMethod( Build_Mode::class, 'get_mode' );
+		$method->setAccessible( true );
+		$get_mode = function ( ...$args ) use ( $method ) {
+			return $method->invoke( null, ...$args );
+		};
```

Comment:
This proxy should no longer be needed since the method will be public.

Reply:
Updated.